src/adapters/kafka/index.ts

```diff
@@ -15,20 +15,17 @@ class KafkaAdapter extends Adapter {
       'kafka'
     )
     const auth: KafkaAuthConfig = await this.getAuthConfig(kafkaOptions?.auth)
-    const securityRequirements = this.AsyncAPIServer.security().map(
-      (sec) => {
-        const secName = Object.keys(sec.values())[0]
-        return this.parsedAsyncAPI.components().securitySchemes().get(secName)
-      }
+    const securityRequirements = this.AsyncAPIServer.json().security.map(
+      (sec) => sec
     )
     const userAndPasswordSecurityReq = securityRequirements.find(
-      (sec) => sec.type() === 'userPassword'
+      (sec) => sec.type === 'userPassword'
```

Comment:
@oviecodes what do you think about extracting these constants inside an enum? then reference it here as something like `SECURITY_TYPE.USER_PASSWORD` or `SECURITY_TYPE.SCRAM_SHA_512` 🤔

Reply:
alright @KhudaDad414 .